apps/health_campaign_field_worker_app/lib/main.dart

```diff
@@ -52,72 +71,84 @@ class MainApplication extends StatelessWidget {
+                        supportedLocales: languages != null
+                            ? languages.map((e) {
+                                final results = e.value.split('_');
 
-                          return results.isNotEmpty
-                              ? Locale(results.first, results.last)
-                              : const Locale('en', 'IN');
-                        })
-                      : [],
-                  localizationsDelegates: [
-                    if (appConfig != null)
-                      AppLocalizations.getDelegate(appConfig),
-                    GlobalWidgetsLocalizations.delegate,
-                    GlobalCupertinoLocalizations.delegate,
-                    GlobalMaterialLocalizations.delegate,
-                  ],
-                  theme: DigitTheme.instance.mobileTheme,
-                  routeInformationParser: appRouter.defaultRouteParser(),
-                  scaffoldMessengerKey: scaffoldMessengerKey,
-                  routerDelegate: AutoRouterDelegate.declarative(
-                    appRouter,
-                    navigatorObservers: () => [AppRouterObserver()],
-                    routes: (handler) => [
-                      if (authState.isAuthenticated)
-                        const AuthenticatedRouteWrapper()
-                      else
-                        const UnauthenticatedRouteWrapper(),
-                    ],
-                  ),
-                ),
-              );
+                                return results.isNotEmpty
+                                    ? Locale(results.first, results.last)
+                                    : const Locale('en', 'MZ');
+                              })
+                            : [const Locale('en', 'MZ')],
+                        localizationsDelegates: [
+                          AppLocalizations.getDelegate(appConfig, isar),
+                          GlobalWidgetsLocalizations.delegate,
+                          GlobalCupertinoLocalizations.delegate,
+                          GlobalMaterialLocalizations.delegate,
+                        ],
+                        theme: DigitTheme.instance.mobileTheme,
+                        routeInformationParser: appRouter.defaultRouteParser(),
+                        scaffoldMessengerKey: scaffoldMessengerKey,
+                        routerDelegate: AutoRouterDelegate.declarative(
+                          appRouter,
+                          navigatorObservers: () => [AppRouterObserver()],
+                          routes: (handler) => [
+                            if (authState.isAuthenticated)
+                              const AuthenticatedRouteWrapper()
+                            else
+                              const UnauthenticatedRouteWrapper(),
+                          ],
+                        ),
+                      ),
+                    )
+                  : const MaterialApp(home: Scaffold(body: Text('Loading')));
```

Comment:
Localize the 'Loading' text

Reply:
@egov-joy  It's a placeholder will be replaced with loader component